packages/bsb/docs/getting-started/guide_components.rst

```diff
@@ -243,7 +243,7 @@ the placement strategy example in :doc:`this section </examples/place_distributi
 
        Familiarize yourself with BSB's CLI.
 
-    .. grid-item-card:: :octicon:`gear;1em;sd-text-warning` Learn about components
+    .. grid-item-card:: :octicon:`gear;1em;sd-text-warning` Domains
```

Comment:
I don't think that components should be replaced by "domains" everywhere, the "Domains" should just be a title that occurs in the sidebar, but this guide is supposed to be a critical guide that teaches people how to make their own components?